src/jit/emitxarch.cpp

```diff
@@ -9185,6 +9191,26 @@ BYTE* emitter::emitOutputRR(BYTE* dst, instrDesc* id)
 
 #endif // _TARGET_AMD64_
     }
+#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)
+    else if ((ins == INS_crc32) || (ins == INS_lzcnt) || (ins == INS_popcnt))
```

Comment:
Beware that if a lot of new instructions will have to be added to the emitter we might have to scramble to fix throughput regressions...

Reply:
I think there would be only 4 instructions in this section (`TZCNT` will be added later).

Reply:
On a related note, I just filed an issue, #14523, as a specialization of #7454, to refactor the emitter and instrdescs to address the cost of all the conditionals.